changes/ticket31684

```diff
@@ -0,0 +1,2 @@
+  o Minor features (onion services):
+    - GETINFO dir/status-vote/current/consensus-microdesc dumps microdesc consensus. Closes ticket 31684.
```

Comment:
Please check how other changes entries look like. They are usually human readable and look like "Implement a new GETINFO command ...".